frontend/public/components/monitoring/_monitoring.scss

```diff
@@ -192,6 +196,19 @@ $tooltip-background-color: #151515;
   width: 100%;
 }
 
+.query-browser__zoom-overlay {
+  background-color: rgba(0, 0, 0, 0.2);
+  bottom: 0;
+  position: absolute;
+  top: 0;
+  z-index: $zindex-modal;
+}
+
+.query-browser__zoom {
```

Comment:
nit: this should go above `__zoom-overlay` if alphabetizing

Reply:
Fixed by https://github.com/openshift/console/pull/2218